mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/ColorChoiceTabBar.kt

```diff
@@ -80,12 +81,18 @@ fun ColorChoiceTabItem(
       contentColor = contentColor,
       shape = shape,
       modifier =
-          modifier.let { if (selected) it.dashedBorder(borderWidth, borderColor, shape) else it }) {
+          modifier
+              .clip(shape)
+              .selectable(selected, onClick = onClick)
+              .then(
+                  if (selected) Modifier.dashedBorder(borderWidth, borderColor, shape)
+                  else Modifier,
+              ),
+  ) {
```

Comment:
[Question] Why was this UI change also part of this pull request?

Reply:
To fix some semantics nodes issues which are required to properly implement the UI tests.

In this case, the selectableGroup Modifier was missing.